build-a-react-native-application-for-ios-and-android-from-start-to-finish/notes/13-make-a-touchable-button-in-react-native.md

```diff
@@ -0,0 +1,24 @@
+# Make a Touchable Button in React Native
+
+**[📹 Video](https://egghead.io/lessons/react-native-make-a-touchable-button-in-react-native)**
+
+## Pressable components
+
+There are many options for wrapping components that should be pressable. The main ones are `<Button />`, `<TouchableOpacity />`, `<TouchableWithoutFeedback />`, `<TouchableHighlight />` and `<Pressable />`. There are varying levels of customizability with each option, however, the [React Native docs](https://reactnative.dev/docs/pressable) now recommend to use the `<Pressable />` component for attaching press handlers to any components, unless you have a really good reason not to.
+
+## New components
```

Comment:
```suggestion
## 🤔  New components
```

Reply:
I put this one on each component individually to remain consistent with the other ones, but let me know if you would prefer it to be on the heading as there are so many new components 👍